index.html

```diff
@@ -1,19 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <img src="img/karma-logo.svg" alt="Karma logo"/>
+      <nav style="height: 71px; bottom: 107px">
+        <ul>
+          <li>Meet Karma</li>
+          <li>How it works</li>
+          <li>Store</li>
+          <li>Blog</li>
+          <li>Help</li>
+          <li>Login</li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="main-section">
+        <h1>Introducing Karma</h1>
+        <h2>Bring WiFi with you, everywhere you go.</h2>
+        <button>Learn more</button>
+      </section>
+      <section class="lower-section">
+        <h2>Everyone needs a little Karma.</h2>
+        <div class="bottom_images">
+          <figure>
+            <img src="/img/icon-devices.svg" alt="devices icon" class="middle-images" />
+            <figcaption>Internet for all devices</figcaption>
```

Comment:
for using the class names, it is better to use some meaningful names which make sense when someone is going through your code, for example, you used a class name "bottom-images" for the div and then for each img tags you used "middle-images" (which does not exist), so it is better to choose 2 different meaningful name which are related to each other, for example for the div you can use class = images-container" and for img tags social-media-images 🙂   Does this make sense?